code/src/components/Dropdown.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import './dropdown.css'
+
+export const Dropdown = (props) => {
+
+    return (
+        <div>
+            <h2 tabIndex={'0'}>{props.question}</h2>
+            <select tabIndex={'0'} onChange={(event) => props.setSelectValue(event.target.value)} value={props.selectValue} required>
```

Comment:
New lines for when an element has more than two attributes. Convention to always start with booleans:
```
<select
  required
  tabIndex={'0'}
  onChange={(event) => props.setSelectValue(event.target.value)}
  value={props.selectValue}>
```